internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -111,10 +111,10 @@ var (
 				Tooling:   &ContainerImageTag{Name: "skuba-tooling", Tag: "0.1.0"},
 			},
 			AddonsVersion: AddonsVersion{
-				Cilium:  &AddonVersion{"1.5.3", 1},
-				Kured:   &AddonVersion{"1.2.0-rev4", 1},
-				Dex:     &AddonVersion{"2.16.0", 4},
-				Gangway: &AddonVersion{"3.1.0-rev4", 3},
+				Cilium:  &AddonVersion{"1.5.3", 2},
+				Kured:   &AddonVersion{"1.2.0-rev4", 2},
+				Dex:     &AddonVersion{"2.16.0", 5},
+				Gangway: &AddonVersion{"3.1.0-rev4", 4},
```

Comment:
IMO, I don't think we need to bump K8s 1.15.2 cluster addons manifest version.

Reply:
I couldn't think of a reason customer installed with K8s 1.15.2 shouldn't also benefit from this fix.

Reply:
From the product perspective.

AFAIK, we don't have to take 4.0.x once 4.1.x released. The customer has to upgrade from 4.0.x to 4.1.x first.

On the other hand, I saw kured bump container image version only on `1.16.2`. Intended or missing?

Reply:
> From the product perspective.
> 
> AFAIK, we don't have to take 4.0.x once 4.1.x released. The customer has to upgrade from 4.0.x to 4.1.x first.

Is upgrade to K8s 1.16.2 is enforced if customer have cluster with 1.15.2? Or could customer upgrade addon only?

> 
> On the other hand, I saw kured bump container image version only on `1.16.2`. Intended or missing?

Reply:
I think the customer could upgrade addon only~

Reply:
Yep, it looks like if would like to benefit for the current cluster w/o cluster upgrade, we should bump addon version for what cluster version we would like to support.

So it should be fine for this change.

https://github.com/SUSE/skuba/blob/c5895c7bfc9fd5fdaa10e6997bb52364b7f69e43/pkg/skuba/actions/addon/upgrade/plan.go#L51